src/jit/morph.cpp

```diff
@@ -6441,17 +6441,17 @@ GenTree* Compiler::fgMorphField(GenTree* tree, MorphAddrContext* mac)
             // Build this tree:  IND(*) #
             //                    |
             //                   ADD(I_IMPL)
-            //                   / \
+            //                   / '\'
             //                  /  CNS(fldOffset)
             //                 /
             //                /
             //               /
             //             IND(I_IMPL) == [Base of this DLL's TLS]
             //              |
             //             ADD(I_IMPL)
-            //             / \
+            //             / '\'
             //            /   CNS(IdValue*4) or MUL
-            //           /                      / \
+            //           /                      / '\'
```

Comment:
This warning is specifically about C language. In C++ context, this sound like a false positive warnig, as language spec says single line comments ignore everything between `//` and EOL characters.

btw, `Ubuntu x64 formatting` leg is failing: https://ci.dot.net/job/dotnet_coreclr/job/master/job/x64_ubuntu_formatting_prtest/28832/consoleFull#1223400814e0cf1bf6-b9e4-4679-b910-ada9d1ab2bcb. This also make these tree diagrams slightly unreadable. How about explicitly supress these warnings with `-Wno-comment` at https://github.com/dotnet/coreclr/blob/721b097db1ac830dafd2590286176ce0e19e84ed/src/pal/tools/gcc-compiler-override.txt#L9

Reply:
This warning is specifically about C language. In C++ context, this sound like a false positive warnig, as language spec says that for a single line comment, compiler shall ignore everything between `//` and EOL characters.

btw, `Ubuntu x64 formatting` leg is failing: https://ci.dot.net/job/dotnet_coreclr/job/master/job/x64_ubuntu_formatting_prtest/28832/consoleFull#1223400814e0cf1bf6-b9e4-4679-b910-ada9d1ab2bcb. This also make these tree diagrams slightly unreadable. How about explicitly supress these warnings with `-Wno-comment` at https://github.com/dotnet/coreclr/blob/721b097db1ac830dafd2590286176ce0e19e84ed/src/pal/tools/gcc-compiler-override.txt#L9